config/jobs/image-pushing/k8s-staging-sig-storage.sh

```diff
@@ -74,3 +74,54 @@ for repo in "${REPOS[@]}"; do
               - .
 EOF
 done
+
+cat >>"${OUTPUT}" <<EOF
+
+# Canary images are used by some Prow jobs to ensure that the upcoming releases
+# of the sidecars work together. We don't promote those canary images.
+# To avoid getting them evicted from the staging area, we have to rebuild
+# them periodically. One additional benefit is that build errors show up
```

Comment:
I'm not sure I understand the comment about build errors showing up early? Shouldn't we see build errors in the post submit job whenever a PR merges?

Reply:
Suppose something changes in the cloud build server or the cloud build image that gets patched into these jobs periodically such that an image build fails. If a repo doesn't merge any PRs for a while, then we won't notice until we try to do a release.